### DISCUSSION TITLE
MDT Implementation

### DISCUSSION THREAD

**Discussion by munchl4xx** [okieselbach/Autopilot-Manager#18] [General]
Is there a way for me to implement this into an MDT build?

We use MDT to deploy our base image with a very small config ready to add to Intune and continue setup via autopilot but so far there is a lot of manual work to get the hash etc.

Any input on how we could achieve this with ease would really help.

**Comment by okieselbach** [comment]
Hi,

sorry for my delay. With the latest enhancements for Automation (https://oliverkieselbach.com/2023/01/24/autopilot-manager-with-additional-automations/) you could run the exe and have an auto approval, but you would still end up in the green screen.
For MDT I would probably look for something like this: https://oliverkieselbach.com/2018/07/17/automation-of-gathering-and-importing-windows-autopilot-information/
This is a very old solution and some of the modules might not work anymore. But I've seen several solutions out there doing the same with newer code. So maybe have a look at something like this. The AutopilotManager is more a manual approach.

**Comment by okieselbach** [comment]
There is a -q parameter in the client exe for quiet mode. You could utilize this.
